Resources/Locale/en-US/ui/options.ftl

```diff
@@ -95,6 +95,7 @@ ui-options-function-open-entity-spawn-window = Open entity spawn menu
 ui-options-function-open-sandbox-window = Open sandbox menu
 ui-options-function-open-tile-spawn-window = Open tile spawn menu
 ui-options-function-open-admin-menu = Open admin menu
+ui-options-function-lock-action-slots = (Un)lock dragging and clearning action slots
```

Comment:
did not fill out the Dutch locale portion, because it's empty anyway xd

Reply:
ye dont bother with the dutch localization, its just testing